<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="baseSize">
   <size>
    <width>800</width>
    <height>600</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>GVD Test App</string>
  </property>
  <widget class="QWidget" name="centralWidget"/>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>20</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu_2">
    <property name="title">
     <string>Вид</string>
    </property>
    <addaction name="actionMapControlPanel"/>
    <addaction name="separator"/>
   </widget>
   <widget class="QMenu" name="menu_3">
    <property name="title">
     <string>Файл</string>
    </property>
    <widget class="QMenu" name="menu_4">
     <property name="title">
      <string>Карта</string>
     </property>
     <addaction name="actionOpenMap"/>
     <addaction name="actionCloseMap"/>
    </widget>
    <widget class="QMenu" name="menu_5">
     <property name="title">
      <string>Фотоплан</string>
     </property>
     <addaction name="actionOpenPhoto"/>
     <addaction name="actionClosePhoto"/>
    </widget>
    <widget class="QMenu" name="menu_6">
     <property name="title">
      <string>Файл маршрута</string>
     </property>
     <addaction name="actionOpenPath"/>
     <addaction name="actionClosePath"/>
    </widget>
    <widget class="QMenu" name="menu_7">
     <property name="title">
      <string>Файл критичных объектов</string>
     </property>
     <addaction name="actionOpenCritSheme"/>
     <addaction name="actionCloseCritSheme"/>
    </widget>
    <addaction name="separator"/>
    <addaction name="menu_4"/>
    <addaction name="menu_5"/>
    <addaction name="menu_6"/>
    <addaction name="menu_7"/>
    <addaction name="separator"/>
    <addaction name="actionClose"/>
   </widget>
   <addaction name="menu_3"/>
   <addaction name="menu_2"/>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionClose">
   <property name="text">
    <string>Выход</string>
   </property>
  </action>
  <action name="actionOpenMapFile">
   <property name="text">
    <string>Открыть</string>
   </property>
  </action>
  <action name="actionCloseMapData">
   <property name="text">
    <string>Закрыть</string>
   </property>
  </action>
  <action name="actionZoomPlus">
   <property name="icon">
    <iconset>
     <normaloff>Resurses/map_zoom_plus24.png</normaloff>Resurses/map_zoom_plus24.png</iconset>
   </property>
   <property name="text">
    <string>Уменьшить масштаб</string>
   </property>
   <property name="toolTip">
    <string>Увелтчить изображение карты</string>
   </property>
   <property name="iconVisibleInMenu">
    <bool>true</bool>
   </property>
  </action>
  <action name="actionZoomMinus">
   <property name="icon">
    <iconset>
     <normaloff>Resurses/map_zoom_minus24.png</normaloff>Resurses/map_zoom_minus24.png</iconset>
   </property>
   <property name="text">
    <string>Увеличить масштаб</string>
   </property>
   <property name="toolTip">
    <string>Уменьшить изображение карты</string>
   </property>
  </action>
  <action name="actionGISParam">
   <property name="text">
    <string>Настройка ГИС-модуля..</string>
   </property>
  </action>
  <action name="actionCreateMatrix">
   <property name="text">
    <string>Создать матрицы...</string>
   </property>
  </action>
  <action name="actionMapControlPanel">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Панель управления картой</string>
   </property>
  </action>
  <action name="actionPathEditor">
   <property name="text">
    <string>Редактор маршрута</string>
   </property>
  </action>
  <action name="actionShowPathProfile">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Профиль по маршруту</string>
   </property>
  </action>
  <action name="actionOpenMap">
   <property name="text">
    <string>Открыть карту</string>
   </property>
  </action>
  <action name="actionCloseMap">
   <property name="text">
    <string>Закрыть карту</string>
   </property>
  </action>
  <action name="actionOpenPhoto">
   <property name="text">
    <string>Открыть фотоплан</string>
   </property>
  </action>
  <action name="actionClosePhoto">
   <property name="text">
    <string>Закрыть фотоплан</string>
   </property>
  </action>
  <action name="actionCritEditor">
   <property name="text">
    <string>Редактор обстановки</string>
   </property>
  </action>
  <action name="actionOpenPath">
   <property name="text">
    <string>Открыть</string>
   </property>
  </action>
  <action name="actionClosePath">
   <property name="text">
    <string>Закрыть</string>
   </property>
  </action>
  <action name="actionOpenCritSheme">
   <property name="text">
    <string>Открыть</string>
   </property>
  </action>
  <action name="actionCloseCritSheme">
   <property name="text">
    <string>Закрыть</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections>
  <connection>
   <sender>actionClose</sender>
   <signal>triggered()</signal>
   <receiver>MainWindow</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>199</x>
     <y>149</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
